WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -71,7 +71,7 @@ import AuthenticationServices
 
     /// Designated Initializer
     ///
-    private init(configuration: WordPressAuthenticatorConfiguration,
+    internal init(configuration: WordPressAuthenticatorConfiguration,
```

Comment:
Nitpick. Since `internal` is the default, I'd omit it.

```suggestion
    init(configuration: WordPressAuthenticatorConfiguration,
```